source/endpoints/jobs.rst

```diff
@@ -0,0 +1,153 @@
+*************
+Jobs
+*************
+
+``GET /jobs/{id}``
+*******************
+
+Gets an existing transcription job by id
+
+**CURL Examples**
+
+.. code:: javascript
+
+  curl -X GET "https://api.rev.ai/revspeech/v1beta/jobs/{id}" -H "Authorization: Bearer <api_key>"
+
+**Parameters**
+
+====================== ===============================================================
+Name                   Description
+====================== ===============================================================
+id ``(required)``      Gets an existing transcription job by id
+====================== ===============================================================
+
+**Responses**
+
+====================== ===============================================================
+Code                   Description
+====================== ===============================================================
+200                    Revspeech API Job
+
+                       ``Example Value``
+
+                       .. code:: javascript
+
+                        {
+                          "id": "111111",
```

Comment:
this needs to match the real data, e.g. the rest of the fields. I would show an example for a successful job and one for a failed job. should also document possible values for enum fields (e.g. possible statuses)

Reply:
This is an example of a real data. If you want the full model, they can go to the models page